t/01-hash.t

```diff
@@ -74,6 +74,14 @@ is_deeply(
     "slice_grep on keys",
 );
 
+my %sgtest = (
+    a => undef,
+    b => 0,
+    c => 5
+);
+my @sgtest = keys %sgtest;
+is_deeply({slice_grep { $_ eq 'b' } \%sgtest, @sgtest}, {b => 0}, "slice_grep over given array",);
```

Comment:
hinzugefügter test mit einem bereits vorhandenem array als @list damit auch der linke teil in 
https://github.com/perl5-utils/Hash-MoreUtils/blob/e724623873c846296e431d4de07d8f4c30caaf38/lib/Hash/MoreUtils.pm#L99
auch durchlaufen wird.